tests/Rule/LatteTemplatesRule/SimpleControl/Fixtures/Blocks/define.latte

```diff
@@ -0,0 +1,13 @@
+{define my-block, \stdClass $paramObject, string $paramString, ?string $paramNullable, $paramNoType, string $paramDefault = 'default', $paramNoTypeDefault = 'default', float $paramFloat = 1.23, int $paramInteger = 123}
+    {php \PhpStan\dumpType($paramObject)}
+    {php \PhpStan\dumpType($paramString)}
+    {php \PhpStan\dumpType($paramNullable)}
+    {php \PhpStan\dumpType($paramNoType)}
+    {php \PhpStan\dumpType($paramDefault)}
+    {php \PhpStan\dumpType($paramNoTypeDefault)}
+    {php \PhpStan\dumpType($paramFloat)}
+    {php \PhpStan\dumpType($paramInteger)}
+{/define}
+
+{include my-block 'paramObject' => $knownObject, 'paramString' => $knownString, 'paramNullable' => null, 'paramNoType' => $knownString, 'paramFloat' => $knownFloat, 'paramInteger' => $knownInteger}
+{include my-block 'paramObject' => $knownString, 'paramString' => $knownInteger}
```

Comment:
@MartinMystikJonas is it possible to call it without keys of parameters? Only include $var1, $var2 …